### ISSUE TITLE
Question: Can the size of the window be adjust post-recording?

### ISSUE THREAD

**Issue by gerhard** [asciinema/agg#12]
I have recorded the same sequence of commands on different hosts.

This one looks great ([apart from that missing emoji](https://github.com/asciinema/agg/issues/2#issuecomment-1237395383)):
![dagger-install-1](https://user-images.githubusercontent.com/3342/188503531-e2d11561-b285-485b-a7c2-e6aca3be7857.gif)

I would like to adjust the size of this window post-recording. Is that possible?
![dagger-install](https://user-images.githubusercontent.com/3342/188503592-35b346a0-7ca1-4a09-9971-aaa3e3951a7e.gif)

FTR:

```sh
agg --version
agg 1.2.0

uname -a
Linux x22 5.15.58 #1-NixOS SMP Fri Jul 29 15:25:34 UTC 2022 x86_64 GNU/Linux

agg --speed=2 --font-size=24 --theme=monokai dagger-install.cast dagger-install-loop.gif
gifsicle --no-loopcount dagger-install-loop.gif > dagger-install.gif
```

Thank you!

---
Attaching the original cast files as zip 👉 [dagger-install-casts.zip](https://github.com/asciinema/agg/files/9491693/dagger-install-casts.zip)

**Comment by sickill** [comment]
Yes, see here https://github.com/asciinema/agg/issues/11#issuecomment-1237238040

I guess it'd be useful to add options for terminal size override, something like `--cols 80 --rows 24`?

**Comment by gerhard** [comment]
> I guess it'd be useful to add options for terminal size override, something like `--cols 80 --rows 24`?

Yes, that would really help.

For now, I manually edited the first line in the cast file and it worked!

<img width="1069" alt="image" src="https://user-images.githubusercontent.com/3342/188511175-b5345bf4-b038-4f2a-b803-9506bad0254e.png">

That answers my question, closing this issue - thank you! 🙌🏻

**Comment by sickill** [comment]
Just released v1.3 with new `--cols / --rows` options.
